app/Classes/Intervention/Transform.php

```diff
@@ -5,8 +5,11 @@
 use App\Enums\ImageFormat;
```

Comment:
whats intervention? should we note that in a comment or will everyone just know that?

Reply:
We mention in the readme that we are using this package for image transformation.